apps/web/app/bookmarks/components/_Content.tsx

```diff
@@ -1,20 +1,29 @@
 'use client';
 
-import { useState, useRef } from 'react';
+import { useState, useRef, useEffect } from 'react';
 import { Content, Menu } from '@social/ui-shared';
 import * as Components from '@/components';
 import { Filter } from '@/components/Filter';
-import { useFilterContext } from '@/contexts';
+import { useFilterContext, useToastContext } from '@/contexts';
 import { useDrawerClickOutside } from '@/hooks/useDrawerClickOutside';
 import { LeftSidebar } from './_LeftSidebar';
 import { RightSidebar } from './_RightSidebar';
 import { BookmarksPage } from '.';
 
 export default function ContentBookmarks() {
   const { layout } = useFilterContext();
+  const { addToast } = useToastContext();
   const [drawerFilterOpen, setDrawerFilterOpen] = useState(false);
   const drawerFilterRef = useRef<HTMLDivElement>(null);
 
+  useEffect(() => {
+    addToast(
+      `Please be aware your bookmarks are stored in a public file connected to your pubky.`,
+      'warning',
+      'Your bookmarks are (semi) public',
```

Comment:
```suggestion
      'Your bookmarks are public',
```

IMO adding `(semi) public` is wrong info, because they are public.